afterpay/src/main/kotlin/com/afterpay/android/internal/AfterpayDrawable.kt

```diff
@@ -0,0 +1,76 @@
+package com.afterpay.android.internal
+
+import androidx.annotation.DrawableRes
+import com.afterpay.android.Afterpay
+import com.afterpay.android.R
+import com.afterpay.android.internal.Locales.EN_AU
+import com.afterpay.android.internal.Locales.EN_CA
+import com.afterpay.android.internal.Locales.EN_GB
+import com.afterpay.android.internal.Locales.EN_NZ
+import com.afterpay.android.internal.Locales.EN_US
+import com.afterpay.android.internal.Locales.ES_ES
+import com.afterpay.android.internal.Locales.FR_CA
+import com.afterpay.android.internal.Locales.FR_FR
+import com.afterpay.android.internal.Locales.IT_IT
+
+private val localeLanguages = mapOf(
+    EN_AU to AfterpayDrawable.EN,
+    EN_GB to AfterpayDrawable.EN_GB,
```

Comment:
What's the difference between `AfterpayDrawable.EN` and `AfterpayDrawable.EN_GB`?

Reply:
If it's just that `EN_GB` uses Clearpay branding and `EN` uses Afterpay, would it be clearer to describe that in the constant? E.g.: `EN_CLEARPAY` and `EN_AFTERPAY`. If these are private/internal properties then I suppose it doesn't matter, we can rename them later with no impact to the public interface.

Reply:
Addressed in [e5d54ce](https://github.com/afterpay/sdk-android/pull/151/commits/e5d54ce22553e75edca4f754b34f71e9d111b4ba)